### ISSUE TITLE
A problem with some custom UI component plugins

### ISSUE THREAD

**Issue by xxdocobxx** [halfnelson/svelte-native#90]
Hi, I am new to svelte-native. Thank you for creating this great project, I love it so much.

I encountered a problem when using some custom UI component plugins like this: [https://github.com/nstudio/nativescript-floatingactionbutton](https://github.com/nstudio/nativescript-floatingactionbutton)

In their usage example, we can use registerElement directly to register the plugin element in Angular and Vue. I don't know if I am not doing it right or other reason, I can't do this in svelte-native.

Currently, I have to create a NativeElementNode for the plugin before using registerElement like this:
```
import { NativeElementNode, registerElement } from 'svelte-native/dom';

registerElement('Fab', () => new NativeElementNode('Fab', require('@nstudio/nativescript-floatingactionbutton').Fab));
```

**Comment by halfnelson** [comment]
Thanks. You are right, it looks like the other frameworks provide a method that does the NativeElementNode for you. If I were to change the behaviour it would break existing code for svelte-native users. 

I will export a new method registerNativeViewElement which does takes a tag name and a method returning a View

**Comment by halfnelson** [comment]
This shipped in 0.6.0 which was just released
